app/konnect/dev-portal/customization/custom.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Customize Kong Developer Portal
+no_version: true
+---
+
+The Kong Developer Portal currently offers three areas you can customize. 
+* [Appearance](#appearance)
+* [Custom URL](#add-a-custom-dev-portal-domain)
+* [Enabling Public Access](/konnect/dev-portal/publish/#access)
+
+This doc covers the details of customizing the appearance of your Developer Portal, and the steps to customize a URL. 
+
+## Appearance
+
+The Dev Portal can be customized by those with Admin roles via the Appearance
+settings in the {{site.konnect_short_name}} Admin UI. To access the Appearance
+settings, click {% konnect_icon dev-portal %} **Dev Portal**, then **Appearance**.
+
+Here, you have the ability to modify the following:
+
+* Logos: default logo and fav icon
+* Home Page Header: welcome message, primary header, and header image
+* Colors: background, text, and button colors
+* Themes: default theme and dark mode
+* Fonts
+
+For details on the requirements for each customizable option, hover over the information (`i`) icon next to each item in the UI.
+
+
+## Custom Developer Portal URL
```

Comment:
Did you mean to delete the separate custom portal URL topic? Either way, any feedback I left on that topic applies here.